pkg/distributor/distributor.go

```diff
@@ -177,6 +180,9 @@ type Config struct {
 
 	// Configuration for forwarding of metrics to alternative ingestion endpoint.
 	Forwarding forwarding.Config
+
+	// Enable the experimental feature to mark metrics as ephemeral.
+	EphemeralSeriesEnabled bool `yaml:"ephemeral_metrics_enabled" category:"experimental"`
```

Comment:
```suggestion
	// Enable the experimental feature to mark series as ephemeral.
	EphemeralSeriesEnabled bool `yaml:"ephemeral_series_enabled" category:"experimental"`
```

Reply:
that's fixed in https://github.com/grafana/mimir/commit/705f50fcb663fa3cd31021a0187f3dda532501c1#diff-8e51742acf8f88fa2ae01855fb19125feb5c54c41023f1eaa8beded9f1b4bcc3